src/automator/tasks/overview.spec.js

```diff
@@ -80,16 +89,47 @@ describe('overview task handler', function() {
     spyOn(fileStore, 'getContent')
       .and.returnValue(q.when(data.fileContent));
     backend.expectPOST('http://richtext/v0/richtext/').respond(201);
-    overview({
-      entity: 'file'
-    }, {
-      entities: {
-        file: data.fileEntity
+    overview(
+      {
+        entity: 'file'
       },
-      collab: data.collab
-    }).then(function(res) {
-      expect(res).toEqual(data.expectedNavItem);
+      {
+        entities: {
+          file: data.fileEntity
+        },
+        collab: data.collab
+      }).then(function(res) {
+        expect(res).toEqual(data.expectedNavItem);
+      });
+    scope.$digest();
+  }));
+
+  it('should update content appId', inject(function() {
+    spyOn(navStore, 'getRoot')
+      .and.returnValue(q.when(data.rootNavItem));
+    spyOn(fileStore, 'getContent')
+      .and.returnValue(q.when(data.fileContent));
+    backend.expectGET('http://collab/v0/extension/?title=My+app+name')
+      .respond(200, data.appResponse);
+    backend.expectPUT('http://collab/v0/collab/1/nav/10/').respond(201, {});
```

Comment:
Minor, but on an update request the status code should be 200 OK, not 201 CREATED